arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -899,6 +890,7 @@
 		F1D5022E245246EC00583CFB /* DisplayUtilityAssociationsViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = F1D5022D245246EC00583CFB /* DisplayUtilityAssociationsViewController.swift */; };
 		F1D5022F24525E2E00583CFB /* DisplayUtilityAssociationsViewController.swift in CopyFiles */ = {isa = PBXBuildFile; fileRef = F1D5022D245246EC00583CFB /* DisplayUtilityAssociationsViewController.swift */; };
 		F1D7508F230730360018B812 /* ElevationViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = F1D7508E230730360018B812 /* ElevationViewController.swift */; };
+		F1D7F6D42792453D004968E3 /* ScottishWildlifeTrust_reserves in Resources */ = {isa = PBXBuildFile; fileRef = F1D7F6D32792453D004968E3 /* ScottishWildlifeTrust_reserves */; };
```

Comment:
Because the shapefile is in this folder, we need to apply the resource tag to it.
```suggestion
		F1D7F6D42792453D004968E3 /* ScottishWildlifeTrust_reserves in Resources */ = {isa = PBXBuildFile; fileRef = F1D7F6D32792453D004968E3 /* ScottishWildlifeTrust_reserves */; settings = {ASSET_TAGS = (scottishWildlifeTrust, ); }; };
```